src/toolchain/toolchain.ts

```diff
@@ -249,7 +269,7 @@ export class SwiftToolchain {
                     return process.env.SDKROOT;
                 }
 
-                return this.getSdkForTarget(DarwinCompatibleTarget.macOS);
+                return this.getSDKPath("macosx");
```

Comment:
I'm assuming we're calling through to something here? Otherwise we should rename this to just `macOS`

Reply:
This is the string that is passed to `xcrun` to get the sdk path. We don't have a darwin target macOS because there is no point setting the SDK for macOS as it is the default